commit 6b4aaf9a6c810be7c696b9edb9ad232a02a83e0b
Author: Carl E. Love <cel@us.ibm.com>
Date:   Mon Dec 19 09:31:13 2016 -0800

    Add support for the IBM POWER 8 NV and NVL variants.
    
    Add support for the IBM POWER 8 NV and NVL variants.
    
    The processor performance counter unit is identical for the various
    variants of the POWER 8 processor.  The variants vary some for the
    non-cpu events.
    
    Signed-off-by: Carl Love <cel@us.ibm.com>

diff --git a/libop/op_cpu_type.c b/libop/op_cpu_type.c
index d03f9d6..0e1f6e4 100644
--- a/libop/op_cpu_type.c
+++ b/libop/op_cpu_type.c
@@ -330,6 +330,11 @@ static op_cpu _get_ppc64_cpu_type(void)
 	if (strncmp(cpu_name_lowercase, "power8e", 7) == 0)
 		cpu_name_lowercase[6] = '\0';
 
+	/* The POWER8NV and POWER8NVL variants have the same core PMU events as
+	 * POWER8. */
+	if (strncmp(cpu_name_lowercase, "power8nv", 8) == 0)
+		cpu_name_lowercase[6] = '\0';
+
 	cpu_type_str[0] = '\0';
 	strcat(cpu_type_str, "ppc64/");
 	strncat(cpu_type_str, cpu_name_lowercase, len);
